hack/cert-generator.sh

```diff
@@ -68,7 +68,7 @@ fi
 set -e
 echo "INFO: Creating CSR: ${csr_name}"
 
-cat <<EOF | kubectl create -f -
+cat <<EOF | kubectl create --validate=false -f -
```

Comment:
Please can you add the comment above.
```suggestion
# signerName is not supported in Kubernetes <= 1.17
cat <<EOF | kubectl create --validate=false -f -
```

Reply:
Makes sense! I did even better, and also added the issue 😛